### DISCUSSION TITLE
Bank Tag categories/groups

### DISCUSSION THREAD

**Discussion by DylanM05** [runelite/runelite#18004] [Ideas]
Would it be possible to add like a "Group" tag like for example:
I have a TOB tag, and in this TOB group I have 3 different tabs that drop down after I click it for melee/range/mage

**Comment by da-loa** [comment]
I know that the Inventory Setups plugin allows this.  But it is slightly different from Bank Tags

**Comment by raiyni** [comment]
Duplicate of https://github.com/runelite/runelite/issues/10963
